### PR TITLE
XDR_playbooks_fix_playbook_inputs

### PR DESCRIPTION
**Pull request by efelmandar**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4046

## Description
Fix XDR playbooks which are using undeclared inputs or declearing unused inputs

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3.yml [hunk 1]
```diff
@@ -1795,12 +1795,6 @@ inputs:
   required: false
   description: Incident ID.
   playbookInputQuery:
-- key: similarIncidentFields
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3.yml:1798]: Why we removed this?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md [hunk 2]
```diff
@@ -5,35 +5,35 @@ This playbook handles false-positive incident closures for Cortex XDR - Malware
...
 ---
 
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
-| Comment | Add a comment to close this incident. | XSOAR Incident #${incident.id} | Optional |
-| Reason | Possible values:<br/> * Unknown <br/>* TruePositive<br/>* FalsePositive | FalsePositive | Optional |
+| Comment | Add comment to close this incident. | XSOAR Incident #${incident.id} | Optional |
+| Reason | Choose From - "Unknown" / "TruePositive" / "FalsePositive" | FalsePositive | Optional |
 | AllowTag | The approving tag name for found indicators. | AllowTag | Optional |
-| AutoUnisolation | Whether automatic un-isolation is allowed. | False | Optional |
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md:29]: So now we are using this input?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md [hunk 3]
```diff
@@ -5,35 +5,35 @@ This playbook handles false-positive incident closures for Cortex XDR - Malware
...
+| AutoUnisolation | Whether automatic unisolation is allowed. | False | Optional |
 | HostID | The ID of the host for running an un-isolation process. | ${incident.deviceid} | Optional |
-| FileSha256 | The file SHA256 you want to block. | ${incident.filesha256} | Optional |
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md:31]: So now we are using this input?

File: Packs/CortexXDR/ReleaseNotes/4_8_18.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+##### Cortex XDR - False Positive Incident Handling
+Fixed an issue with playbook inputs.
+##### Cortex XDR incident handling v3
+Fixed an issue with playbook inputs. 
+##### JOB - Cortex XDR query endpoint device control violations
+Added a missing playbook input.
```
Comment (altmannyarden) [inline Packs/CortexXDR/ReleaseNotes/4_8_18.md:8]: Added missing playbook inputs*

File: Packs/PANWComprehensiveInvestigation/ReleaseNotes/1_3_11.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Palo Alto Networks - Endpoint Malware Investigation v3
+Fixed an issue with a task using wrong playbook input.
```
Comment (altmannyarden) [inline Packs/PANWComprehensiveInvestigation/ReleaseNotes/1_3_11.md:4]: lets specify the task

File: Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3.yml (hunk 1 above)
Comment (efelmandar) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3.yml:1798]: another unused playbook input

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md (hunk 2 above)
Comment (efelmandar) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md:29]: yes it used in the "Auto-Unisolate" task

**Pull request merged**